s3/replication/manager/src/s3replicationmanager/__main__.py

```diff
@@ -17,7 +17,17 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+import argparse
 from s3replicationmanager.app import ReplicationManagerApp
 
 if __name__ == '__main__':
-    ReplicationManagerApp().run()
\ No newline at end of file
+    # create parser object
+    parser = argparse.ArgumentParser(description='''Replicator server help''')
+
+    # adding an arguments
+    parser.add_argument('--configfile', type=str, metavar='path', help='config yaml file with  absolutepath')
```

Comment:
help='Path to replication manager configuration file (format: yaml).'

Reply:
done